server/etcdserver/api/etcdhttp/health.go

```diff
@@ -99,30 +103,62 @@ func NewHealthHandler(lg *zap.Logger, hfunc func(ctx context.Context, excludedAl
 }
 
 var (
+	// Deprecated: use rootHealthCheckCounter instead.
 	healthSuccess = prometheus.NewCounter(prometheus.CounterOpts{
 		Namespace: "etcd",
 		Subsystem: "server",
 		Name:      "health_success",
-		Help:      "The total number of successful health checks",
+		Help:      "The total number of successful health checks (Deprecated, use etcd_server_root_health_total instead)",
 	})
+	// Deprecated: use rootHealthCheckCounter instead.
 	healthFailed = prometheus.NewCounter(prometheus.CounterOpts{
 		Namespace: "etcd",
 		Subsystem: "server",
 		Name:      "health_failures",
-		Help:      "The total number of failed health checks",
+		Help:      "The total number of failed health checks (Deprecated, use etcd_server_root_health_total instead)",
 	})
+	rootHealthCheckCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
same concern here.

Reply:
I think the accum counter can not be easily be derived from individual checks. The sum of all the checks depends on the number of checks there is.

Reply:
I agree with Han, this is redundant with the other counter.

Reply:
moved the root counts to healthCheckCounter with `name="/"`